### ISSUE TITLE
Should we swap images of wizard and rune knight?

### ISSUE THREAD

**Issue by WeaverThree** [atmafox/FFRPG4e#177]

**Comment by atmafox** [comment]
Yeah, rune knight's image is clearly a red mage.  Not sure the wizard image fits for rune knight but looking at the abilities you get from it, perhaps so.

**Comment by atmafox** [comment]
consensus seems to be that while rune knight's image is a red mage, the wizard image doesn't fit for rune knight.  I say we push this off until we do an art sprint?

**Comment by BrunoCarvalhoPaula** [comment]
Sure, lets push this off to after 4.x
